batch-setup/cron.py

```diff
@@ -529,6 +529,8 @@ def is_power_of_two(x):
                         '<planet-url>.md5 convention to specify an md5 '
                         'file for the OSM planet file specified by the '
                         'planet-url argument.')
+    parser.add_argument('--skip-post-import-steps', dest='run_post_import_steps', action='store_false')
+    parser.set_defaults(run_post_import_steps=True)
```

Comment:
`action='store_false'` will set default to `True` for you. [see doc](https://docs.python.org/2/library/argparse.html#action)

```suggestion
    parser.add_argument('--skip-post-import-steps', dest='run_post_import_steps', action='store_false')
```

Reply:
oh interesting.  So I can just leave that out.  Thx!

Reply:
That is a fascinating bit of user friendliness.